static/src/javascripts/projects/common/modules/discussion/comments.js

```diff
@@ -422,20 +423,36 @@ Comments.prototype.reportComment = function(e) {
                 comment = form.elements.comment.value;
 
             if (category.value !== '0') {
-                DiscussionApi.reportComment(commentId, {
+                DiscussionApi
+                    .reportComment(commentId, {
                     emailAddress: form.elements.email.value,
                     categoryId: category.value,
                     reason: comment
-                });
+                })
+                .then(self.reportCommentSuccess.bind(self, form), self.reportCommentFailure.bind(self) );
             }
-
-            bonzo(form).addClass('u-h');
         });
     }).appendTo(
         $('#comment-' + commentId + ' .js-report-comment-container').first()
     ).removeClass('u-h');
 };
 
+Comments.prototype.reportCommentSuccess = function(form) {
+    bonzo(form).addClass('u-h');
+};
+
+Comments.prototype.reportCommentFailure = function() {
+    var message = 'Sorry there was an error reporting this comment. Please try another browser or network connection.',
+        error = bonzo.create(
+        '<div class="d-discussion__error ' + this.getClass('error', true) + '">' +
```

Comment:
@piuccio Yeah could do. For this, i was just using the same functionality as we do for posting comments. But that's equally as do-able. What you think?